FAQ.qmd

```diff
@@ -123,6 +123,13 @@ title : "Frequently Asked Questions"
     </p>
 </details>
 
+<details>
+  <summary>Availability of compressed EO data (zipped)</summary>
```

Comment:
@abuiquoc ,IMHO, I think we should rephrase it as a question that user can relate to? Such as: Is it possible to acquire data with compression? 

What do you think?

Reply:
Yes, this will fit better in the FAQ section. I changed it to a question.

Reply:
@abuiquoc. Thank you I will go forward with the merge